client/me/account/main.jsx

```diff
@@ -272,7 +301,8 @@ const Account = React.createClass( {
 				<FormLegend>{ translate( 'Holiday Snow' ) }</FormLegend>
 				<FormLabel>
 					<FormCheckbox
-						checkedLink={ this.valueLink( 'holidaysnow' ) }
+						checked={ this.getUserSetting( 'holidaysnow' ) }
```

Comment:
I think it's not needed - the `checked` setter on the DOM element accepts any value and converts it to boolean. And the `holidaysnow` setting is pretty much guaranteed to be a boolean already.